### ISSUE TITLE
Question about base image. Why Ubuntu with Node installing manually insted of Node?

### ISSUE THREAD

**Issue by gustawdaniel** [jshimko/meteor-launchpad#96]
Hi.

I would like to ask why you do not base on node image but on Ubuntu image. Are there some dependencies that Ubuntu image has but Node image has not?

In my opinion building image basing on Node is more consistent with docker philosophy of installing minimal number of packages. What do you think about this?

**Comment by ghost** [comment]
Yes, I'd like to know why too. It takes a long time for me to build a quite simple app.

**Comment by maasha** [comment]
Maybe because the official Node image is based on Alpine Linux and not Ubuntu. IMHO, one should always use a Debian stable base image (and that would most certainly also require a Node update).

**Comment by auterium** [comment]
@maasha not all images of node are based on alpine. There are at least 2 viable image set options:
- node:8.15.1-jessie, node:8.15-jessie, node:8-jessie, node:carbon-jessie <= based on debian:jessie
- node:8.15.1-stretch, node:8.15-stretch, node:8-stretch, node:carbon-stretch, node:8.15.1, node:8.15, node:8, node:carbon <= based on debian:stretch

@gustawdaniel Is this ever going to be consider?
